draft-reddy-ipsecme-ikev2-pqc-auth.md

```diff
@@ -131,6 +132,53 @@ SLH-DSA does not introduce a new hardness assumption beyond those inherent to th
 
 # Signature Algorithm Use and Hashing in IKEv2 with ML-DSA and SLH-DSA
 
+For integrating ML-DSA and SLH-DSA into IKEv2, we take the approach used in [RFC8420]
+
+The implementation MUST send a SIGNATURE_HASH_ALGORITHMS notify with an Identity" (5) hash function.
+ML-DSA and SLH-DSA are only defined with the "Identity" hash and MUST NOT be sent to a receiver that has not indicated support for the "Identity" hash.
+
+When generating a signature with ML-DSA or SLH-DSA, the IKEv2 implementation would take the InitiatorSignedOctets string or the ResponderSignedOctets string (as appropriate), logically send it to the identity hash (which leaves it unchanged), and then pass it into the ML-DSA or SLH-DSA signer as the message to be signed (along with the fixed context string "IKEv2 Auth" (`49 4b 45 76 32 20 41 55 54 48`).
+The resulting signature is placed into the The resulting signature is the Signature Value.
```

Comment:
The line looks incorrect, I think you mean "The resulting signature is placed into the Signature Value".

Reply:
Oops, careless copy and paste.  Fixed